app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::GreetingsController < ApplicationController
+  # def index
+  #     @greetings = Greeting.all
+
+  #     render json: @greetings, status: :ok
+  #   end
```

Comment:
- [ ]  Kindly note that it is generally nad practice to push code that is commented out. It makes your code base look very messy and cluttered. KIndly consider removing this block of code that  is commented out.